podcasts/BookmarkManager.swift

```diff
@@ -78,6 +84,30 @@ class BookmarkManager {
         }
     }
 
+    // MARK: - Playback
+
+    /// Play the bookmark
+    func play(_ bookmark: Bookmark) {
```

Comment:
I thought about where to put this, but this made the most sense since it's central and specific to bookmarks.

Reply:
Hmmm, that's indeed a tricky one. However, I wonder if `PlaybackManager` wouldn't be the correct place to have this.

Perhaps, a new method like `PlaybackManager.play(episodeUuid: String, time: TimeInterval)`. This method knows a lot about `PlaybackManager`, which I think is a signal maybe this might belong there.

Also, by doing this you can get rid of the `DataManager` from here.

Thoughts?